Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -169,7 +169,7 @@ MaybeLoaderError FlacLoaderPlugin::load_picture(FlacRawMetadataBlock& block)
     // Note: We are seeking before reading the value to ensure that we stayed inside buffer's size.
     auto offset_before_seeking = memory_stream.offset();
     LOADER_TRY(memory_stream.seek(mime_string_length, SeekMode::FromCurrentPosition));
-    picture.mime_string = { block.data.bytes().data() + offset_before_seeking, (size_t)mime_string_length };
+    picture.mime_string = StringView { block.data.bytes().data() + offset_before_seeking, (size_t)mime_string_length };
```

Comment:
`picture.mime_string` will now invalid after the next block is loaded.

Reply:
Would this be a more acceptable replacement?

```cpp
// With picture.mime_string as String
picture.mime_string = TRY(String::from_utf8({ block.data.bytes().data() + offset_before_seeking, (size_t)mime_string_length }));
```

Reply:
Yep, I think that'd be fine :slightly_smiling_face:  (I assume mine strings are utf8)